src/dispatch_package/dispatch.cpp

```diff
@@ -76,21 +75,37 @@ struct ClosestDepotInfo {
 
 
 struct Parameters {
-  kilowatt_hours battery_cap_kwh        = 200.0_kWh;       //kW*hr
-  kWh_per_km     kwh_per_km             = 1.2_kWh_per_km;  //kW*hr/km
-  double         charging_rate          = 150;             //kW
-  double         search_radius          = 1000;            //m //TODO
-  double         zstops_frac_stopped_at = 0.2;
-  double         zstops_average_time    = 10;              //seconds
+  kilowatt_hours battery_cap_kwh       = 200.0_kWh;
+  kWh_per_km     kwh_per_km            = 1.2_kWh_per_km;
+  dollars        bus_cost              = 500'000.0_dollars;
+  dollars        battery_cost_per_kwh  = 100.0_dollars;
+  dollars        depot_charger_cost    = 50'000.0_dollars;
+  kilowatts      depot_charger_rate    = 125.0_kW;
+  dollars        nondepot_charger_cost = 600'000.0_dollars;
+  kilowatts      nondepot_charger_rate = 500.0_kW;
+  int32_t        chargers_per_depot    = 1;
```

Comment:
this either needs to be an abstraction (there would certainly be >1 charger per depot, so 1 charger, its cost being equal to the number of chargers there would actually be) or we need to add lower power chargers in proportion to the bus capacity of the depot, perhaps.